### PR TITLE
[MCP2020A] Space Charge + 3 Drift Window Configs

### PR DESCRIPTION
**Pull request by gputnam**
For all of the event selection samples we need space charge and 3 drift windows both enabled.

I have added in a base config to enable the space charge services. I have also added in every config for g4/detsim/reco needed with both settings on.

### CODE REVIEW

**Comment by wesketchum** [comment]
Is any of this duplicated with #48 ?

**Comment by gputnam** [comment]
uhhh yeah looks like one of the files is -- reco2_3drift_windows_sce.fcl

File: sbndcode/JobConfigurations/standard/detsim/detsim_3drift_windows_sce.fcl [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#include "standard_detsim_sbnd.fcl"
+
+#include "3drift_services_sbnd.fcl"
+#include "enable_spacecharge_services_sbnd.fcl"
```
Comment (absolution1) [inline sbndcode/JobConfigurations/standard/detsim/detsim_3drift_windows_sce.fcl:4]: Probably a dumb Q but does anything in the detsim chain need access to the SCE service?

File: sbndcode/JobConfigurations/standard/reco/reco1_3drift_windows_sce.fcl [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#include "standard_reco1_sbnd.fcl"
+
+#include "3drift_services_sbnd.fcl"
+#include "enable_spacecharge_services_sbnd.fcl"
```
Comment (absolution1) [inline sbndcode/JobConfigurations/standard/reco/reco1_3drift_windows_sce.fcl:4]: Is the SCE service needed for reco1?

**Review by absolution1 — COMMENTED** [review]
Thanks @gputnam looks nice!
Had a couple of minor questions.

**Comment by absolution1** [comment]
> uhhh yeah looks like one of the files is -- reco2_3drift_windows_sce.fcl

Hey, I did warn you that I was committing the reco2 FCLS.  I figured the PR had already been merged.
I guess we have to figure out the ordering.
I guess that @ascarff could merge #48 and then this PR is updated to keep the history in order.

File: sbndcode/JobConfigurations/standard/detsim/detsim_3drift_windows_sce.fcl (hunk 1 above)
Comment (gputnam) [reply sbndcode/JobConfigurations/standard/detsim/detsim_3drift_windows_sce.fcl:4]: I don't know, but I added it in to be on the safe side. I figured there was no downside in being consistent.

File: sbndcode/JobConfigurations/standard/reco/reco1_3drift_windows_sce.fcl (hunk 2 above)
Comment (gputnam) [reply sbndcode/JobConfigurations/standard/reco/reco1_3drift_windows_sce.fcl:4]: Same as for detsim.

**Comment by gputnam** [comment]
@absolution1 yeah that sounds good to me!

File: sbndcode/JobConfigurations/standard/detsim/detsim_3drift_windows_sce.fcl (hunk 1 above)
Comment (absolution1) [reply sbndcode/JobConfigurations/standard/detsim/detsim_3drift_windows_sce.fcl:4]: Ah ok, sure. 
There is a bit of a downside here, in that there's extra points of maintenance.
Personally, I would say that, if `gen -> g4_sce -> detsim -> reco1 -> reco2_sce` runs/works then that would probably be better.  But, I'd obvs defer the decision to you / @jzennamo / @wesketchum

**Pull request merged**